### PR TITLE
Adds helpers to html build step

### PR DESCRIPTION
**Pull request by richsilv**
Any js/json files in the _helpers_ directory will be available in the locals passed to Jade templates.

### CODE REVIEW

**Comment by alanshaw** [comment]
:eyeglasses:

File: build/jade.js [hunk 1]
```diff
@@ -5,6 +5,9 @@ var find = require('find')
 var jade = require('jade')
 var async = require('async')
 var mkdirp = require('mkdirp')
+var extend = require('extend')
+var requireDir = require('require-dir')
+var helpers = requireDir('../helpers')
 var md = require('markdown-it')({
```
Comment (alanshaw) [inline build/jade.js]: I think markdown-it can come out of here now and become a helper...

File: build/jade.js [hunk 2]
```diff
@@ -5,6 +5,9 @@ var find = require('find')
 var jade = require('jade')
 var async = require('async')
 var mkdirp = require('mkdirp')
+var extend = require('extend')
+var requireDir = require('require-dir')
+var helpers = requireDir('../helpers')
```
Comment (alanshaw) [inline build/jade.js]: What happens if this directory doesn't exist?

To be safe, can we use `path.join(__dirname, '..', 'helpers')` here instead?

**Comment by alanshaw** [comment]
Please also resolve the conflict while you're there ;)

File: build/jade.js (hunk 2 above)
Comment (richsilv) [inline build/jade.js]: I don't really understand this.  Surely the result of `path.join(__dirname, '..', 'helpers')` could just as well not exist?

Comment (alanshaw) [inline build/jade.js]: Sorry those are separate points in the same comment.

I was worried about the relative path being affected by `process.cwd()`, but looking into it the require-dir module [handles it](https://github.com/aseemk/requireDir/blob/master/index.js#L12) so don't worry about that second comment.

We should handle the directory not existing though.

**Pull request merged**